virttest/env_process.py

```diff
@@ -1026,6 +1027,17 @@ def preprocess(test, params, env):
             env["cpu_model"] = cpu.get_qemu_best_cpu_model(params)
         params["cpu_model"] = env.get("cpu_model")
 
+    if not params.get("vcpu_driver"):
+        cpu_model = params.get("cpu_model")
+        if cpu_model:
+            qemu_path = utils_misc.get_qemu_binary(params)
+            for vcpu_driver in utils_qemu.get_supported_devices_list(qemu_path,
+                                                                     "CPU"):
+                if re.match(r"%s-\w+-cpu" % cpu_model, vcpu_driver):
```

Comment:
We can have a function for the convenience, e.g.
```python
# utils_qemu.py
def is_supported_cpu_device(model_name):
    for vcpu_driver in get_supported_devices_list(...):
        if ...:
            return True
    return False
```

Reply:
Use model_name to check is okay for me, but return `True/False` seems that not suitable. I need to use the returned cpu_driver after.
So can I add a function like `find_supported_device(bin_path, pattern, category=None)` to find the most matching one?